src/OpenTelemetry.Exporter.Prometheus/PrometheusExporterMetricsHttpServer.cs

```
@@ -112,27 +113,91 @@ private void WorkerThread()
+                                            {
+                                                var metricValueBuilder = builder.AddValue();
+                                                metricValueBuilder = metricValueBuilder.WithValue(doubleValue);
+                                                metricValueBuilder.WithLabel(label.Key, label.Value);
+                                            }
+
+                                            builder.Write(writer);
+                                            break;
+                                        }
+
+                                    case AggregationType.LONGSUM:
+                                        {
+                                            // This cannot occcur as we are iterating Double metrics.
+                                            break;
+                                        }
+
+                                    case AggregationType.SUMMARY:
+                                        {
+                                            // Not supported yet.
+                                            break;
+                                        }
+                                }
+                            }
+
+                            foreach (var metric in this.exporter.GetAndClearLongMetrics())
+                            {
+                                var labels = metric.Labels;
+                                var builder = new PrometheusMetricBuilder()
+                                    .WithName(metric.MetricName)
+                                    .WithDescription(metric.MetricDescription);
+
+                                switch (metric.AggregationType)
+                                {
+                                    case AggregationType.DOUBLESUM:
+                                        {
+                                            // This cannot occcur as we are iterating Long metrics.
```

please report an error here